interface/forms/gad7/gad7.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * gad-7.inc - common includes and constants for the gad-7 form
+ * version 1.0.0  July 2020
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ * @copyright Copyright (c) 2021 ruth moulton ruth@muswell.me.uk
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once("../../globals.php");
+ require_once("$srcdir/api.inc");
+
+// menu strings
+$str_default = 'Please select an answer';
+$str_not = 'Not at all';
+$str_several = 'Several days';
+$str_more = 'More than half the days';
+$str_nearly = 'Nearly every day';
+$str_somewhat = 'Somewhat difficult';
+$str_very = 'Very difficult';
+$str_extremely = 'Extremely difficult';
+
+$str_nosave_exit = "Close form without saving answers";
+$str_nosave_confirm = "Are you sure you'd like to quit without saving your answers?";
+
+$str_form_name = "General Anxiety Disorder 7 (GAD-7)";
+$str_form_title = "GAD-7";
+
+$str_q8 = xlt('How difficult have these problems made it to do work, take care of things at home, or get along with other people? ');
+$str_q8_2 = xlt('(This question is optional and not included in the final score)');
```

Comment:
better to translate all of these string, but not escape them:
```
$str_default = xl('Please select an answer');
$str_not = xl('Not at all');
$str_several = xl('Several days');
$str_more = xl('More than half the days');
$str_nearly = xl('Nearly every day');
$str_somewhat = xl('Somewhat difficult');
$str_very = xl('Very difficult');
$str_extremely = xl('Extremely difficult');

$str_nosave_exit = xl("Close form without saving answers");
$str_nosave_confirm = xl("Are you sure you'd like to quit without saving your answers?");

$str_form_name = xl("General Anxiety Disorder 7 (GAD-7)");
$str_form_title = xl("GAD-7");

$str_q8 = xl('How difficult have these problems made it to do work, take care of things at home, or get along with other people?');
$str_q8_2 = '(' . xl('This question is optional and not included in the final score') . ')';
```

Reply:
To explain above:
1. The translation pipeline can  only automatically find new constants (that get placed in the translation spreadsheet  for translating) if they are within a translation function call, so better to translate the string rather than a variable.
2. Better to only escape when it is outputted to html. in this case just setting a variable. When we output this variable to html, then can escape it there (ie. so use xl() here instead of xla/t() )